src/math/ceil.c

```diff
@@ -0,0 +1,5 @@
+int
+ceil (double x)
+{
+    return (int)(x + 1);
```

Comment:
missing `(` and `)` around return value.

also if x == (int)x then ceil return (int)x